components/collective-page/CollectiveNotificationBar.js

```diff
@@ -98,6 +98,9 @@ const messages = defineMessages({
 });
 
 const getNotification = (intl, status, collective, host, LoggedInUser, refetch) => {
+  const numberOfAdmins = [CollectiveType.EVENT, CollectiveType.PROJECT].includes(collective.type)
+    ? collective?.parentCollective?.coreContributors?.filter(c => c.isAdmin)?.length
```

Comment:
These also include `MEMBER`.
We can either add the `admin` property to this query or update the graphqlV1 members resolver to return inherited admins.